resources/ory/charts/hydra/templates/deployment.yaml

```diff
@@ -35,8 +35,15 @@ spec:
             - /bin/sh
             - -c
             - |
-              export DB_URL=$(echo $DSN | cut -d '@' -f2 | cut -d ':' -f1)
-              export DB_PORT=$(echo $DSN | cut -d '@' -f2 | cut -d ':' -f2 | cut -d '/' -f1)
+              DB_URL_PORT=$(echo $DSN | cut -d '@' -f2 | cut -d '/' -f 1 )
+
+              if echo "$DB_URL_PORT" | grep -q ':'; then
```

Comment:
could you add just simple comment here for the future, like in which flow how example DSN looks like and why we split. It will be easier for the future in case of debugging  :)